cli/src/main/java/org/infinispan/cli/connection/rest/RestConnection.java

```diff
@@ -287,7 +288,17 @@ public Collection<String> getAvailableSites(String container, String cache) thro
 
    @Override
    public Iterable<String> getCacheKeys(String container, String cache) throws IOException {
-      return new IterableJsonReader(parseBody(fetch(() -> client.cache(cache).keys()), InputStream.class), s -> s == null || "_value".equals(s));
+      return new IterableJsonReader(parseBody(fetch(() -> client.cache(cache).keys()), InputStream.class), Function.identity());
```

Comment:
should the key be wrapped in a JSON object?
Currently, it is returning
```
[pedro-laptop-4-31499@cluster//containers/default/caches/dist]> ls -f FULL  -m 2
{key: "key-770", value: "abc", timeToLiveSeconds: "-1", maxIdleTimeSeconds: "-1", created: "-1", lastUsed: "-1", expireTime: "-1"}
{key: "key-227", value: "abc", timeToLiveSeconds: "-1", maxIdleTimeSeconds: "-1", created: "-1", lastUsed: "-1", expireTime: "-1"}
[pedro-laptop-4-31499@cluster//containers/default/caches/dist]> ls -f VALUES -m 2
{key: "key-770", value: "abc"}
{key: "key-227", value: "abc"}
[pedro-laptop-4-31499@cluster//containers/default/caches/dist]> ls -m 2
key-770
key-227
```
and I'm wondering if it makes sense to return
```
[pedro-laptop-4-31499@cluster//containers/default/caches/dist]> ls -m 2
{key: "key-770"}
{key: "key-227"}
```

Reply:
BTW, it looks like `IterableJsonReader` uses `Function.identity()` everywhere. Would it make sense to remove it?

Reply:
The `Function` being passed in is indeed a relic and I'll remove it

Reply:
or you can truncate the key/value to fit a fixed column size.
but I would do the JSON format for now and add the other ones later if requested.